openshift/e2e-tests-openshift.sh

```diff
@@ -21,18 +20,6 @@ readonly TARGET_IMAGE_PREFIX="$INTERNAL_REGISTRY/$SERVING_NAMESPACE/knative-serv
 
 env
 
-function enable_admission_webhooks(){
-  header "Enabling admission webhooks"
-  add_current_user_to_etc_passwd
-  disable_strict_host_checking
```

Comment:
We can now also remove these two functions: add_current_user_to_etc_passwd, disable_strict_host_checking. And the SSH_PRIVATE_KEY variable. And we should also remove https://github.com/openshift/knative-serving/blob/master/openshift/admission-webhooks.yaml because it is not used anymore..